### DISCUSSION TITLE
Hardware recommendations

### DISCUSSION THREAD

**Discussion by jancoow** [blakeblackshear/frigate#4734] [Ask A Question]
I've 6 reolink 420 cameras, which I want to try out with frigate!

I'm looking for a good NVR setup, which is energy efficient and not to expensive. What hardware should I be looking at?> 

Currently a coral is nowhere available so I guess it needs the cpu power for now.

**Comment by NickM-27** [comment]
You'll want something either an intel iGPU that supports QSV or an amd CPU that also has an iGPU (assuming that energy efficiency is your goal).

If energy efficiency isn't as big of a deal then could build your own server (what I have done) and put your own GPU and a more efficient CPU.

**Comment by jancoow** [comment]
I was looking at a HP EliteDesk 800 G2 DM 35W, for small form factor and cheap available second hand. Not sure if that supports QSV. 

It has Intel-HD-Graphics-530

But maybe I can better go for an I3 or celeron? Say for example a system with intel Core i3 6100
